package.json

```diff
@@ -10,7 +10,7 @@
         "consumer": "node ./queue-consumer.js"
     },
     "dependencies": {
-        "ably": "ably/ably-js#0.9.0-beta.6",
+        "ably": "ably/ably-js#1.0.13",
```

Comment:
Now we don't need to use a prerelease we can use the npm version, like you did [here](https://github.com/ably/demo-mobile-phonegap-cordova/pull/10/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R11), we don't need to point to a github tag